spec/lib/declare_schema/field_spec_spec.rb

```diff
@@ -55,6 +55,15 @@
           expect(subject.schema_attributes(col_spec)).to eq(type: :string, limit: 100, null: true)
         end
       end
+
+      it 'raises error when DEFAULT_STRING_LIMIT option is nil when not explicitly set in field spec' do
```

Comment:
I don't think `DEFAULT_STRING_LIMIT` should be capitalized here. It makes it seem like a constant.